bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tarray.py

```diff
@@ -0,0 +1,23 @@
+# Author: Enric Tejedor CERN  11/2018
+
+################################################################################
+# Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from ROOT import pythonization
+
+from ._generic import add_len
+
+@pythonization()
+def pythonize_tarray(klass, name):
+    # Parameters:
+    # klass: class to be pythonized
+    # name: string containing the name of the class
+
+    if name == 'TArray':
```

Comment:
This will propagate to TH1, but it can be fine

Reply:
Yes I think so too, if `TH1` inherits from `TArray` it means the histogram can also be seen as an array, in which case having a len method is fine.